client/components/button/test/index.js

```diff
@@ -0,0 +1,1064 @@
+/**
+ * Created by nitrajka on 09/10/16.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { expect } from 'chai';
+import { shallow } from 'enzyme';
+
+/**
+ * Internal dependencies
+ */
+import Button from '../';
+import Gridicon from 'components/gridicon';
+
+describe( 'Button', function() {
+	describe( 'not compact', function() {
+		describe( 'button tag', function() {
+			const text = 'Text';
+			const icon = ( <Gridicon size={ 18 } icon="arrow-left" /> );
+			const disabled = 'Disabled';
+
+			describe( 'basic styled', function() {
+				it( 'renders with text', function() {
+					const button = shallow( <Button>{ text }</Button> );
+					expect( button ).to.contain( 'Text' );
+					expect( button.is( '.button' ) ).to.equal( true );
```

Comment:
`chai-enzyme` gives us a [`className` assertion](https://github.com/producthunt/chai-enzyme#classnamestr), which would read more clearly:

```js
expect( button ).to.have.className( 'button' );
```

As an aside, Chai has a [`.true` assertion](http://chaijs.com/api/bdd/#method_true) too which could be used in place of yours here:

```js
expect( button.is( '.button' ) ).to.be.true;
```